### PR TITLE
server: respond to translations commands

### PR DESCRIPTION
**Pull request by chris-ramon**
#### Overview
- Issue: https://github.com/connectedcars/connectedcars/issues/305
- [x] Make bender respond to import/export translations.

#### Test plan
- [ ] Manual test

#### Action required
- n/a

### CODE REVIEW

**Comment by kevinsimper** [comment]
LGTM 👍

**Comment by chris-ramon** [comment]
squashing git commits for better reviewing into single commit: https://github.com/connectedcars/bender/pull/13/commits/436cc1eeb8def0be3637b199fff34b08b7b1401a

File: server.js [hunk 1]
```diff
@@ -59,3 +59,29 @@ bender.hears('msg ([^\\s]+) (.*)', 'direct_message', (bot, message) => {
...
   })
 })
+
+bender.hear('export translations', (bot, message) => {
+  bot.reply(message,'Exporting translations..')
+  exec('echo "sudo docker-compose -f /deploy/docker-compose-prod.yml -p kevinsimper run web node exporttranslations.js" | gcloud compute ssh docker-1 --zone europe-west1-b --', (error, stdout, stderr) => {
+    console.log(stdout)
+    console.error(stderr)
+    console.error({error})
+    if (error)
```
Comment (kevinsimper) [inline server.js]: Be aware of these with missing parentheses as they can cause unwanted behaviours 😄

Comment (chris-ramon) [inline server.js]: good catch!, I'll be adding parentheses on new PR.

**Pull request merged**